bids2openminds/main.py

```diff
@@ -312,11 +312,63 @@ def create_subjects(subject_id, layout_df, layout, collection):
     return subjects_dict, subject_state_dict, subjects_list
 
 
+def create_file_bundle(path, collection, parent_file_bundle=None):
```

Comment:
this seems to run on the full path with no exceptions which means that the dataset folder itself is registered as a file bundle which it should not.

reminder: if you store the openminds results within the dataset directory the code would also register this folder as a filebundle (if run second time) which it should not